features/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/domain/SessionReplayRequestFactoryTest.kt

```diff
@@ -1,12 +1,12 @@
-/*
+ /*
  * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
  * This product includes software developed at Datadog (https://www.datadoghq.com/).
  * Copyright 2016-Present Datadog, Inc.
  */
 
-package com.datadog.android.sessionreplay.internal.domain
+ package com.datadog.android.sessionreplay.internal.domain
```

Comment:
bad formatting in this file still occurs.

Reply:
hmmm I started to hate these tools honestly 🤦